masu/external/downloader/downloader_interface.py

```diff
@@ -24,15 +24,15 @@ class DownloaderInterface(ABC):
     """Masu interface definition to download cost usage reports."""
 
     @abstractmethod
-    def download_current_report(self):
+    def download_reports(self, number_of_months=1):
         """
-        Download the current cost usage report.
+        Download cost usage reports.
 
         Implemented by a downloader class.  Must return a list of
         full path of CUR files that are part of the cost usage report.
 
         Args:
-            None
+            (Int) number_of_months - number of prior months to download.
 
         Returns:
             (List) List of full local file paths to CUR files.
```

Comment:
Any thought to just define the method here and let subclasses redefine. I know its an interface, just seems like some copy & paste happening here that could potentially be avoided.

Or maybe put it in `masu/external/report_downloader.py`?